BrowserKit/Sources/Common/Utilities/Notifiable.swift

```diff
@@ -29,7 +29,7 @@ extension NotificationCenter: NotificationProtocol {
                          queue: queue,
                          using: block)
     }
-}
+} 
```

Comment:
There's an extra space after the }, let's remove it
```suggestion
}
```